// Data for linear operators

syntax = "proto3";

message Data {
  enum Type {
    DENSE_MATRIX = 0;
    SPARSE_MATRIX = 1;
  }
  Type data_type = 1;
  bytes value = 2;

  int32 m = 3;
  int32 n = 4;

  // SPARSE_MATRIX
  int32 nnz = 5;
}
